mashup-example/using-build-tool/qlik-sense-desktop/configure.js

```diff
@@ -25,15 +25,6 @@ const nuked = embed.createConfiguration({
     theme: 'myTheme',
     language: 'en-US',
     keyboardNavigation: true,
-    constraints: {
```

Comment:
These were the same as the default values, so don't think it need to be specified